src/rpc/rpcquorums.cpp

```diff
@@ -360,7 +363,17 @@ UniValue quorum_sigs_cmd(const JSONRPCRequest& request)
     uint256 msgHash = ParseHashV(request.params[3], "msgHash");
 
     if (cmd == "sign") {
-        return llmq::quorumSigningManager->AsyncSignIfMember(llmqType, id, msgHash);
+        if (request.params.size() > 4) {
```

Comment:
I believe it is correct to use isNull() here
```suggestion
        if (!request.params[4].isNull()) {
```